R/process_extract_alarms_telecare.R

```diff
@@ -20,6 +20,12 @@ process_extract_alarms_telecare <- function(
   # Check that the supplied year is in the correct format
   year <- check_year_format(year)
 
+  # Check that we have data for this year
+  if (!check_year_valid(year, "CH")) {
```

Comment:
```suggestion
  if (!check_year_valid(year, "AT")) {
```
Do you mean AT in this script?